itl/plugins-contrib.d/vmware.conf

```diff
@@ -237,9 +237,12 @@ object CheckCommand "vmware-esx-dc-runtime-tools" {
 		"--alertonly" = {
 			set_if = "$vmware_alertonly$"
 		}
-		"--open-vm-tools" = {
+		"--open_vm_tools_ok" = {
```

Comment:
Why is this renamed? What happened to `--open-vm-tools`? Will this change break anything?

Reply:
It has been renamed in upstream.